src/Http/Http.Abstractions/src/WebSocketManager.cs

```diff
@@ -42,4 +45,21 @@ public virtual Task<WebSocket> AcceptWebSocketAsync()
     /// <param name="acceptContext"></param>
     /// <returns></returns>
     public virtual Task<WebSocket> AcceptWebSocketAsync(WebSocketAcceptContext acceptContext) => throw new NotImplementedException();
+
+    private string DebuggerToString()
+    {
+        return IsWebSocketRequest switch
+        {
+            false => "IsWebSocketRequest = false",
+            true => $"IsWebSocketRequest = true, RequestedProtocols = {string.Join(",", WebSocketRequestedProtocols)}",
```

Comment:
Pascal case is how other booleans are written:
```suggestion
            false => "IsWebSocketRequest = False",
            true => $"IsWebSocketRequest = True, RequestedProtocols = {string.Join(",", WebSocketRequestedProtocols)}",
```